src/Bucket/bucket_parameterizations.jl

```diff
@@ -50,24 +50,6 @@ function partition_surface_fluxes(
     return (; F_melt = F_melt, F_into_snow = F_into_snow, G = G)
 end
 
-"""
-    surface_albedo(α_sfc::FT, α_snow::FT, σS::FT, σS_c::FT)::FT where {FT <: AbstractFloat}
-
-Returns the bulk surface albedo, linearly interpolating between the albedo
-of snow and of the surface, based on the snow water equivalent S relative to
-the parameter S_c.
-
-The linear interpolation is taken from Lague et al 2019.
-"""
-function surface_albedo(
```

Comment:
Why was this function moved to `Bucket.jl`? What should we put in `Bucket.jl` vs `bucket_parameterizations.jl`?

Reply:
after discussion, we'll move these over to parameterizations